src/frontend/screens/Observation/ObservationView.js

```diff
@@ -86,68 +145,57 @@ class ObservationView extends React.Component<ODVProps> {
     return (
       <ScrollView style={styles.container}>
         <>
-          <Image
-            source={require("../../images/category-pin.png")}
-            style={styles.categoryPin}
-          />
-          <View style={styles.categoryIconContainer}>
-            <CategoryIcon iconId={(preset || {}).icon} />
-          </View>
-          {lat != null && lon != null && (
-            <View style={{ flexDirection: "row", alignSelf: "center" }}>
-              <FormattedCoords
-                lon={lon}
-                lat={lat}
-                style={styles.positionText}
-              />
+          {lat !== null && lon !== null && (
```

Comment:
I _think_ `lat` and `lon` can be `undefined` in some situations. That's why I did `lat != null` (which also catches `undefined`) instead of `lat !== null` (which only catches `null`). It's obscure JS, so we should probably have a comment about it.

Reply:
@gmaclennan if it can be undefined or null we can just use `!lat` to catch that

Reply:
Then we can't have anything at [Null Island](https://en.wikipedia.org/wiki/Null_Island) ;) I've been trying to avoid `!variable` in the code, because it can lead to unexpected bugs.